superset-frontend/src/setup/setupApp.ts

```diff
@@ -22,8 +22,16 @@ import { SupersetClient } from '@superset-ui/core';
 import {
   getClientErrorObject,
   ClientErrorObject,
-} from '../utils/getClientErrorObject';
-import setupErrorMessages from './setupErrorMessages';
+} from 'src/utils/getClientErrorObject';
+import setupErrorMessages from 'src/setup/setupErrorMessages';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+declare global {
+  interface Window {
+    $: any;
+    jQuery: any;
+  }
+}
```

Comment:
is this related to the feature flag change? i didn't see how

Reply:
It's not, but these globals used to be declared in the `featureFlags.ts` file. I moved them closer to where they are actually used.